problems/1.types+operators.js

```diff
@@ -2,15 +2,20 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
+function isNumber(any) {
+  return typeOf (any) && "val"
+
```

Comment:
close typeof any === "number" 

make sure to spell things with proper casing. YOu need to use the equality operatory not the AND to check if the type is anumber. AND checks for truthyness on both sides.